addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2958,6 +2958,22 @@ options.registry.HideFooter = VisibilityPageOptionUpdate.extend({
     shownValue: 'shown',
 });
 
+options.registry.ContainerWidthFooter = options.registry.ContainerWidth.extend({
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+    /**
+     * Allow to customize the view only if the footer copyright section
+     * is the target (due to the data-apply-to)
+     * @override
+     */
```

Comment:
Add blank lines: before the first //---, after the second //--- and before the `@override` please :pray: 

Also, details, but our convention is to start documentation of function by a conjugated verb: `Allows`. Also `.` at the end of your sentence.